packages/did-auth-siop-op-authenticator/src/session/OpSession.ts

```diff
@@ -239,16 +279,16 @@ export class OpSession {
       .withExpiresIn(expiresIn)
 
       .addDidMethod(didMethod)
-      .suppliedSignature(SuppliedSigner(keyRef, context, this.getKeyAlgorithm(keyRef.type)), identifier.did, identifier.controllerKeyId)
-      .registrationBy(SIOP.PassBy.VALUE)
-      .response(SIOP.ResponseMode.POST)
+      .suppliedSignature(SuppliedSigner(keyRef, context, this.getKeyAlgorithm(keyRef.type)), identifier.did, identifier.controllerKeyId, this.getKeyAlgorithm(keyRef.type))
+      .registration({
+        registrationBy: {
+          passBy: PassBy.VALUE
```

Comment:
This looks of. Either it is incorrect in the SIOP package or incorrect here

Reply:
I am looking into it. But what should the value be ?

Reply:
Seems correct to me.

Reply:
It should by .registration: { passby:xxxx} directly

Reply:
For record: this was discussed in person. So, either it should be changed in SIOP package or it should be fine like this.